advanced_alchemy/base.py

```diff
@@ -334,73 +350,148 @@ def create_registry(
 
 
 class UUIDBase(UUIDPrimaryKey, CommonTableAttributes, DeclarativeBase):
-    """Base for all SQLAlchemy declarative models with UUID primary keys."""
+    """Base for all SQLAlchemy declarative models with UUID primary keys.
+
+    .. seealso::
+        :class:`UUIDPrimaryKey`
+        :class:`CommonTableAttributes`
+        :class:`DeclarativeBase`
+    """
 
     registry = orm_registry
 
 
 class UUIDAuditBase(CommonTableAttributes, UUIDPrimaryKey, AuditColumns, DeclarativeBase):
-    """Base for declarative models with UUID primary keys and audit columns."""
+    """Base for declarative models with UUID primary keys and audit columns.
+
+    .. seealso::
+        :class:`CommonTableAttributes`
+        :class:`UUIDPrimaryKey`
+        :class:`AuditColumns`
+        :class:`DeclarativeBase`
+    """
 
     registry = orm_registry
 
 
 class UUIDv6Base(UUIDv6PrimaryKey, CommonTableAttributes, DeclarativeBase):
-    """Base for all SQLAlchemy declarative models with UUID primary keys."""
+    """Base for all SQLAlchemy declarative models with UUID primary keys.
```

Comment:
would be better if this and others (`UUIDvXBase` and `UUIDvXAuditBase`) can mention the UUID version in the docstring